src/Anemone/Foreign/Mempool.hs

```diff
@@ -47,6 +48,9 @@ foreign import ccall unsafe "anemone_mempool_free"
 foreign import ccall unsafe "anemone_mempool_total_alloc_size"
   totalAllocSize :: Mempool -> IO Int64
 
+foreign import ccall unsafe "anemone_is_pointer_aligned"
+  isPointerAligned :: Ptr Void -> IO Bool
```

Comment:
Is there a conversion function from `CBool -> Bool` ? `CBool` makes it a bit hard to use.

I could just use `CInt` and then compare to `0`.